agent/Makefile

```diff
@@ -10,7 +10,7 @@ override QUBES_CFLAGS := -I. -I../libqrexec -g -O2 -Wall -Wextra -Werror \
    -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fPIC -std=gnu11 -D_POSIX_C_SOURCE=200809L \
 	-D_GNU_SOURCE $(CFLAGS)
 override LDFLAGS += -pie -Wl,-z,relro,-z,now -L../libqrexec
-override LDLIBS=-lqrexec-utils $(shell pkg-config --libs $(VCHAN_PKG)) -lpam
```

Comment:
`-lpam` should be added in this Makefile, not elsewhere. Maybe a better way is to do it conditionally on `-DHAVE_PAM` in `$(CFLAGS)`? Or even separate make variable (`HAVE_PAM`?) to set both `-DHAVE_PAM` in `CFLAGS` and `-lpam` in `LDFLAGS`.